frontend/src/reviews/ReviewComments.tsx

```diff
@@ -31,23 +45,107 @@ export default function ReviewComments({ release, accessRequest }: ReviewComment
+    }
+    decisionsAndComments.sort(sortByCreatedAtAscending)
+    return decisionsAndComments.map((response) => {
+      if (isReviewResponse(response)) {
+        return <ReviewDecisionDisplay key={response.createdAt} response={response as ReviewResponse} />
+      } else {
+        return <ReviewCommentDisplay key={response.createdAt} response={response as ReviewComment} />
+      }
+    })
+  }, [reviews, release, accessRequest])
+
+  async function submitReviewComment() {
+    setCommentSubmissionError('')
+    if (!newReviewComment) {
+      setCommentSubmissionError('Please provide a comment before submitting.')
+      return
+    }
+    if (currentUser) {
+      const newComment: ReviewComment = {
+        comment: newReviewComment,
+        user: currentUser.dn,
+        createdAt: new Date().toISOString(),
+      }
+      if (release) {
+        const updatedRelease: ReleaseInterface = release
+        if (!updatedRelease.comments) {
+          updatedRelease.comments = []
+        }
+        updatedRelease.comments.push(newComment)
+        const res = await putRelease(updatedRelease)
+        if (res.ok) {
+          mutateRelease()
+          setNewReviewComment('')
+        } else {
+          setCommentSubmissionError(await res.json())
+        }
+      } else if (accessRequest) {
+        const updatedAccessRequest: AccessRequestInterface = accessRequest
+        if (!updatedAccessRequest.comments) {
+          updatedAccessRequest.comments = []
+        }
+        updatedAccessRequest.comments.push(newComment)
+        const res = await submitAccessRequestComment(
+          accessRequest.modelId,
+          accessRequest.id,
+          updatedAccessRequest.comments,
+        )
+        if (res.ok) {
+          mutateAccessRequest()
+          setNewReviewComment('')
+        } else {
+          setCommentSubmissionError(await res.json())
+        }
+      } else {
+        setCommentSubmissionError('There was a problem submitting this comment, please try again later.')
+      }
+    } else {
+      setCommentSubmissionError('There was a fetching user information, please try again later.')
+    }
+  }
 
   if (isReviewsError) {
     return <MessageAlert message={isReviewsError.info.message} severity='error' />
   }
 
+  if (isCurrentUserError) {
+    return <MessageAlert message={isCurrentUserError.info.message} severity='error' />
+  }
+
   return (
     <>
       {reviews.length > 0 && <Divider />}
-      {isReviewsLoading && <Loading />}
+      {isReviewsLoading && isCurrentUserLoading && <Loading />}
       {reviewDetails}
+      <>
+        <Stack spacing={1} justifyContent='center' alignItems='flex-end'>
+          <TextField
+            size='small'
+            sx={{ width: '100%' }}
+            placeholder='Add a comment'
+            value={newReviewComment}
+            onChange={(e) => setNewReviewComment(e.target.value)}
+          ></TextField>
```

Comment:
Can use `<TextField />` instead of `<TextField></TextField>` as you're not providing any children